Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGridPreparePass.cpp

```diff
@@ -44,7 +44,7 @@ namespace AZ
         {
             // load shader
             // Note: the shader may not be available on all platforms
-            AZStd::string shaderFilePath = "Shaders/DiffuseGlobalIllumination/DiffuseProbeGridPrepare.azshader";
+            AZStd::string shaderFilePath = "shaders/diffuseglobalillumination/diffuseprobegridprepare.azshader";
```

Comment:
Can we make a constant for this at the top of the namespace within this file?
i.e
```c++
constexpr AZStd::string_view DiffuseProbeGridPrepareShader = "shaders/diffuseglobalillumination/diffuseprobegridprepare.azshader";
```